pyproject.toml

```diff
@@ -23,14 +23,16 @@ dynamic = ["version"]
 
 dependencies = [
     # Core dependencies
-    "DLite-Python>=0.4.5,<1",
-    "numpy>=1.21,<3",
+    "DLite-Python>=0.4.5,<0.5.28",
+    "numpy>=1.21,<2", # To increase to numpy2 with tripper=0.3.5
+    # The reason for this is that pint is not updated to numpy2
+    # but also does not specify that it depends numpy1
```

Comment:
```suggestion
    "numpy>=1.21,<3", # it fails now because of numpy issues 
    # will be fixed with tripper being updated, which updates pint whih updates numpy
```